okhttp/src/main/java/okhttp3/Response.java

```diff
@@ -301,6 +309,53 @@ public long receivedResponseAtMillis() {
         + '}';
   }
 
+
+  /**
+   * Enum for status code class (family).
+   * It is intended to provide convenient method for switch-case branching on status class (family),
+   * instead of HTTP code calculations within client code.
+   */
+  public enum StatusFamily {
```

Comment:
Spec uses "class" https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html.

Reply:
Yes, but "class" seems already be meaningful in java :)